src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Cart/ApplyCouponToCartHandlerSpec.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\CommandHandler\Cart;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\ApiBundle\Command\Cart\ApplyCouponToCart;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\PromotionCouponInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Order\Processor\OrderProcessorInterface;
+use Sylius\Component\Promotion\Repository\PromotionCouponRepositoryInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+final class ApplyCouponToCartHandlerSpec extends ObjectBehavior
+{
+    function let(
+        OrderRepositoryInterface $orderRepository,
+        PromotionCouponRepositoryInterface $promotionCouponRepository,
+        OrderProcessorInterface $orderProcessor
+    ) {
+        $this->beConstructedWith($orderRepository, $promotionCouponRepository, $orderProcessor);
+    }
+
+    function it_is_a_message_handler(): void
+    {
+        $this->shouldImplement(MessageHandlerInterface::class);
+    }
+
+    function it_applies_coupon_to_cart(
+        OrderRepositoryInterface $orderRepository,
+        OrderInterface $cart,
+        PromotionCouponRepositoryInterface $promotionCouponRepository,
+        PromotionCouponInterface $promotion,
+        OrderProcessorInterface $orderProcessor
```

Comment:
Arguments from `let` method should be placed as the first one in specs
```suggestion
        OrderRepositoryInterface $orderRepository,
        PromotionCouponRepositoryInterface $promotionCouponRepository,
        OrderProcessorInterface $orderProcessor,
        OrderInterface $cart,
        PromotionCouponInterface $promotion
```